src/Compilers/CSharp/Test/Emit2/Semantics/CollectionExpressionTests.cs

```diff
@@ -34722,7 +34722,18 @@ static void Main()
                 }
                 """;
 
-            var verifier = CompileAndVerify([source, s_collectionExtensions], expectedOutput: "[1, 2, 3],", targetFramework: targetFramework, verify: Verification.Skipped);
+            var expectedOutputString = "[1, 2, 3],";
+            var verifier = CompileAndVerify(
+                [source, s_collectionExtensions],
+                expectedOutput: targetFramework switch
```

Comment:
Perhaps simplify the `targetFramework` checks for `expectedOutput` and `VerifyIL()` to use `targetFramework == TargetFramework.Standard` rather than `targetFramework switch ...`.

Reply:
My original motivation for `switch` was to hard-fail if test is modified and not all cases are handled, but checking for `TargetFramework.Standard` seems to fulfill this purpose as well